codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionEncoder.java

```diff
@@ -331,22 +331,26 @@ private Http2Stream requireStream(int streamId) {
      */
     private final class FlowControlledData extends FlowControlledBase {
         private final CoalescingBufferQueue queue;
+        private int dataSize;
 
         public FlowControlledData(Http2Stream stream, ByteBuf buf, int padding, boolean endOfStream,
                                    ChannelPromise promise) {
             super(stream, padding, endOfStream, promise);
-            queue = new CoalescingBufferQueue(promise.channel());
+            queue = new CoalescingBufferQueue(promise.channel(), 8);
```

Comment:
Yeah I'd be in favor of going lower ... maybe default to 4?  Should we just make that the default in `CoalescingBufferQueue`?

@louiscryan thoughts?